### PR TITLE
Add GenoFLU 1.03

### PR DESCRIPTION
**Pull request by sage-wright**
This PR adds the GenoFLU tool, which was made by the USDA to provide whole-genome genotypes for flu genomes.

Pull Request (PR) checklist:
- [x] Include a description of what is in this pull request in this message.
- [x] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [x] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - ~(optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)~
- [x] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [x] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [x] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [x] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [x] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

File: genoflu/1.03/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    wget \
+    ca-certificates \
+    procps && \
+    apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+# install genoflu via bioconda; install into 'base' conda env
+# genoflu is not recognized in micromamba by the version tag
+RUN micromamba install --yes --name base --channel conda-forge --channel bioconda \
+    genoflu=${GENOFLU_VER} && \
+    micromamba clean --all --yes
```
Comment (Kincekara) [inline genoflu/1.03/Dockerfile:30]: micromamba clean --all --force-pkgs-dirs --yes

**Comment by Kincekara** [comment]
Thank you for your PR @sage-wright. It looks great!
I will request a minor change. Could you try to add `-f` or `--force-pkgs-dirs` to `micromamba clean`? It helps to reduce the final image size.

**Comment by sage-wright** [comment]
@Kincekara, thanks for the suggestion! I've added it

**Review by Kincekara — APPROVED** [review]

**Comment by Kincekara** [comment]
Thank you for your contribution!
You can check the image deployment here: https://github.com/StaPH-B/docker-builds/actions/runs/9893545830.

**Pull request merged**